plugins/Process.cpp

```diff
@@ -63,6 +63,54 @@ procptr_t Process::virtualFunction(const procptr_t classObject, const size_t ind
 	return peekPtr(vTable + (index * m_pointerSize));
 }
 
+procptr_t Process::findPattern(const std::vector< uint8_t > &pattern, const Module &module) {
+	for (const auto &region : module.regions()) {
+		if (!region.readable) {
+			continue;
+		}
+
+		const auto ret = findPattern(pattern, region.address, region.size);
+		if (ret) {
+			return ret;
+		}
+	}
+
+	return 0;
+}
+
+procptr_t Process::findPattern(const std::vector< uint8_t > &pattern, procptr_t address, const size_t size) {
+	std::vector< uint8_t > buffer(32768);
+	const auto chunks = size / buffer.size();
```

Comment:
I assume the "rounding down" is intentional here (In cases where `size % buffer.size() != 0`)?

Reply:
Yes, the remaining space is handled later in the function.